### PR TITLE
Handle volume risk multiplier thoughtfully when kissing

### PR DESCRIPTION
**Pull request by jy2wong**
When kissing, take the worse of the kissing and volume multipliers. In
particular, this means that

* there is no bonus for not talking when you are kissing
* kissing and loud talking multipliers are not multiplied together when
  kissing

### CODE REVIEW

File: src/data/calculate.ts [hunk 1]
```diff
@@ -251,21 +251,27 @@ export const calculateActivityRisk = (data: CalculatorData): number | null => {
     ): number => (given === '' ? 1 : table[given].multiplier)
 
     let effectiveDuration = data.duration
-    multiplier *= mulFor(Distance, data.distance)
     if (data.distance === 'intimate') {
```
Comment (beshaya) [inline src/data/calculate.ts:254]: This would probably be easier to parse as a switch statement.

**Review by beshaya — CHANGES_REQUESTED** [review]
Please add a test case or two in data/__tests__/calculate.test.ts

**Comment by blanchardjeremy** [comment]
I guess I'm curious how there is such an activity as "kissing + loud talking" ... It seems like it should really be two activities: 1 that you log as kissing and the other that you log as loud talking. I'm curious what you have in mind when you say that you could select both. What's the example scenario?

Thanks!

**Comment by jy2wong** [comment]
> I guess I'm curious how there is such an activity as "kissing + loud talking" ... It seems like it should really be two activities: 1 that you log as kissing and the other that you log as loud talking. I'm curious what you have in mind when you say that you could select both. What's the example scenario?
> 
> Thanks!

I didn't have one in mind to start with (just wanted to address #620), but if pressed, I'd suggest a karaoke night date.

**Comment by beshaya** [comment]
It's not so much that there's an activity where you *should* select both as
the calculator *lets* you select both. Possibly we should just remove the
talking dropdown?

**Comment by beshaya** [comment]
Catherio approved giving kissing the same multiplier as loud talking, if you want to include that here (ok in a follow-up as well)

**Comment by blanchardjeremy** [comment]
My two cents: I'm in favor of removing the conversation volume when selecting kissing. And I'll let y'all make a final call.

**Comment by jy2wong** [comment]
I removed the conversational volume question in the kissing scenario, but then the only precaution left is the indoor/outdoor/ventilation one, which doesn't even do anything (it just tells you that it's ignoring the value).

What about hiding the *entire* Precautions section if you're kissing?

**Comment by blanchardjeremy** [comment]
That sounds right to me. I'd suggest we remove the warning about "outdoor doesn't apply when kissing" and re-add some notice that explicitly says "no precautions apply when kissing" ... so the user understands why everything else disappeared. 

Thoughts?

(Maybe just removing them is obvious enough)

**Comment by jy2wong** [comment]
I don't think it's necessary to explain. I think new additions to the bare minimum reading path need to achieve/satisfy at least one of these:

* address surprise (make it easy to quickly answer natural questions)
* be a sufficiently common source of surprise to justify making everyone else read it
* drive helpful changes in behaviour
* build future intuition about how to model

I don't think kissing satisfies any of these.

**Comment by beshaya** [comment]
Makes sense to me. The reason this was here in the first place was because
we didn't have kissing, only "cuddling" or "close" where the other
multipliers make sense.

**Comment by jy2wong** [comment]
Current version complete removes precautions section including the header during the kissing scenario and reworks translation string to specifically be about the close case and not the intimate case. Also made it show up for the other ventilation-related bonuses.

PTAL?

File: src/locales/en.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
       "label": "Precautions",
       "environment_header": "Environment",
       "environment_question": "Is it indoor or outdoor? Is the space ventilated, or is the air filtered?",
-      "no_intimate_bonus_outdoors": "Due to very close distances, we are not confident that being outdoors reduces the risk in a substantial way. Thus, we are not providing any bonus for being outdoors when intimate.",
+      "no_close_bonus_environment": "Good environment choices are being ignored in the calculation. We are not confident that good ventilation or being outdoors strongly reduces the risk when people are this close to each other.",
```
Comment (beshaya) [inline src/locales/en.json:16]: Kind of awkward wording?

"Good things are being ignored"

Also, changing the json key will cause all existing translations to show the English until they are re-translated. I think it is preferable to show the current translations (keep "no_intimate_bonus_outdoors")

**Review by beshaya — CHANGES_REQUESTED** [review]

**Review by beshaya — APPROVED** [review]

**Pull request merged**